tensorflow/python/keras/layers/convolutional.py

```diff
@@ -418,6 +418,8 @@ class Conv1D(Conv):
       Specifying any stride value != 1 is incompatible with specifying
       any `dilation_rate` value != 1.
     padding: One of `"valid"`, `"causal"` or `"same"` (case-insensitive).
+      `"valid"` means no padding. `"same"` results in padding the input such
```

Comment:
Let's make this more specific? `same` results in padding evenly to the left/right or up/down of the input such that output has the same height / width dimension as the input.
Same for all the following.

Reply:
Nice, yes will add this

Reply:
What about

`"same" results in padding the input such that the output has the same spatial size as the original input`

Reply:
I have made the suggested changes, making this more specific, I also corrected this in all the places that could find that it was applicable to do so. I think this will help users understand how to use padding correctly when using TensorFlow.